### PR TITLE
Feat: created default page for collection

### PR DESCRIPTION
**Pull request by isha-dubey**
## Description
Added display options of add  collection and add  request, along with a sparrow infographic to guide the user. After adding a collection, the user should be prompted to name the collection and add a description. Once the collection is created, an option to create a new API request within the collection should be provided.

Figma:

## What type of PR is this? (check all applicable)

- [x] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build

## Have you tested locally?

- [x] 👍 yes
- [ ] 🙅 no, because I am lazy

## Added tests?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md

### CODE REVIEW

**Comment by punitkashyup** [comment]
Please attach the issue link

File: src/packages/@workspaces/features/workspace-default-page/layout/WorkspaceDefaultPage.svelte [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+        showImportCollectionPopup();
+        }}>
+  
+          <div class="d-flex justify-content-center align-items-center" style="height: 79px"> 
+              <img src="{libraryAdd}" alt=""/>
+          </div>  
+          <div class="d-flex justify-content-center align-items-center" style="height: 41px; background-color:#22232E; padding:10px; font-size:14px; " >
+            Add Collection
+          </div>
+      </div>
```
Comment (itsmdasifraza) [inline src/packages/@workspaces/features/workspace-default-page/layout/WorkspaceDefaultPage.svelte:28]: don't use hard code colors,  bootstrap classes not used

File: src/packages/@workspaces/features/workspace-default-page/layout/WorkspaceDefaultPage.svelte [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+
+  <!-- style="
+    display: flex; 
+    justify-content: center; 
+    align-items: center; 
+    height: 100%; 
+    font-size: 2em; 
+    color: #fff; 
+    cursor: pointer; -->
```
Comment (itsmdasifraza) [inline src/packages/@workspaces/features/workspace-default-page/layout/WorkspaceDefaultPage.svelte:60]: unused comment

File: src/packages/@app/pages/Collections/CollectionsPage.svelte [hunk 3]
```diff
@@ -242,6 +243,13 @@
...
           </Motion>
+      
+        {:else}
+        <Motion {...scaleMotionProps} let:motion>
+      <WorkspaceDefaultPage 
+      showImportCollectionPopup={() => (isImportCollectionPopup = true)}
+      onItemCreated={_viewModel.handleCreateItem} />
+        </Motion>
         {/if}
       {/if}
```
Comment (itsmdasifraza) [inline src/packages/@app/pages/Collections/CollectionsPage.svelte:254]: code not formatted

**Comment by punitkashyup** [comment]
Please attach the issue link

**Comment by punitkashyup** [comment]
Please attach the issue link

**Comment by punitkashyup** [comment]
Please attach the issue link

**Comment by punitkashyup** [comment]
Please attach the issue link

**Review by itsmdasifraza — APPROVED** [review]

**Pull request merged**